### ISSUE TITLE
New version cannot be published

### ISSUE THREAD

**Issue by cart** [cart/glsl-to-spirv#12]
I just tried publishing the new `glsl-to-spirv` version, but its blocked on the fact that it now uses a "two crate" structure. We moved the "big" files into a new crate, which doesn't have the increased crate size limit. We have options:

1. Move the big files back into the original crate
2. Ask the crates.io team nicely to also raise the limit on the new crate.
3. Try to make `naga` work for our needs (either by using the subset of glsl currently supported or moving to wgsl)

**Comment by enfipy** [comment]
Hey, any news from the crates.io team? Or it's obsolete due to naga and renderer changes?

**Comment by cart** [comment]
Yup they bumped our limit a few weeks ago, but I think theres a good chance that glsl-to-spirv will be removed in the next release (largely hinges on whether or not we choose to adopt wgsl). If we decide to stick with glsl for the next release (and naga doesn't get first class glsl support in time), we can publish a new version of this crate. Just trying to choose my battles here to keep redundant work to a minimum :)
